extensions/resource-deployment/notebooks/docker/2017/deploy-sql2017-image.ipynb

```diff
@@ -32,19 +41,43 @@
         },
         {
             "cell_type": "code",
-            "source": "def run_command():\r\n    print(\"Executing: \" + cmd)\r\n    !{cmd}\r\n    if _exit_code != 0:\r\n        raise SystemExit(f'Shell command:\\n\\n\\t{cmd}\\n\\nreturned non-zero exit code: ' + str(_exit_code) + '.\\n')\r\n    print(f'Successfully executed: {cmd}')\r\n\r\ncmd = 'docker version'\r\nrun_command()",
+            "source": [
+                "import pandas,sys,getpass,os,json,html,time\r\n",
+                "pandas_version = pandas.__version__.split('.')\r\n",
+                "pandas_major = int(pandas_version[0])\r\n",
+                "pandas_minor = int(pandas_version[1])\r\n",
+                "pandas_patch = int(pandas_version[2])\r\n",
+                "if not (pandas_major > 0 or (pandas_major == 0 and pandas_minor > 24) or (pandas_major == 0 and pandas_minor == 24 and pandas_patch >= 2)):\r\n",
+                "    sys.exit('Please upgrade the Notebook dependency before you can proceed, you can do it by running the \"Reinstall Notebook dependencies\" command in Azure Data Studio.')\r\n",
+                "\r\n",
```

Comment:
Should we open a bug where line ends should be standardized to \n? I think this will cause issues cross-plat and is caused by Chris's change

Reply:
https://github.com/microsoft/azuredatastudio/issues/6604